test/utils.js

```diff
@@ -63,14 +63,13 @@ function getBlock(b) {
 }
 
 function lmsrMarginalPrice(funding, netOutcomeTokensSold, outcomeIndex) {
-    const b = Decimal(funding.valueOf()).div(netOutcomeTokensSold.length).ln()
-
-    return Decimal(netOutcomeTokensSold[outcomeIndex].valueOf()).div(b).exp().div(
-        netOutcomeTokensSold.reduce(
-            (acc, tokensSold) => acc.add(Decimal(tokensSold.valueOf()).div(b).exp()),
-            Decimal(0)
-        )
-    ).valueOf()
+    const b = new Decimal(funding.toString()).div(Decimal.ln(netOutcomeTokensSold.length))
```

Comment:
This liquidity parameter was calculated wrong (see the offset parenthesis)! That was the main reason those skipped tests were failing :cry: